tests/create_open_test.go

```diff
@@ -33,143 +34,143 @@ func TestCreateOpen(t *testing.T) {
 
 	ipfs := testingCoreAPI(t, node)
 
-	Convey("orbit-db - Create & Open", t, FailureHalts, func(c C) {
+	t.Run("orbit-db - Create & Open", func(t *testing.T) {
 		dbPath, clean := testingTempDir(t, "db")
 		defer clean()
 
 		orbit, err := orbitdb.NewOrbitDB(ctx, ipfs, &orbitdb.NewOrbitDBOptions{Directory: &dbPath})
-		c.So(err, ShouldBeNil)
+		assert.NoError(t, err)
```

Comment:
If i remember correctly, asset.NoError won't stop, you should make something like

```golang
If !assert.NoError(t, err) {
    return
}
```

Reply:
I made an helper in berty/berty, look for `func checkErr(t *testing.T, err error)`